presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -678,9 +678,15 @@ protected RowExpression visitInPredicate(InPredicate node, Void context)
         {
             ImmutableList.Builder<RowExpression> arguments = ImmutableList.builder();
             arguments.add(process(node.getValue(), context));
-            InListExpression values = (InListExpression) node.getValueList();
-            for (Expression value : values.getValues()) {
-                arguments.add(process(value, context));
+            // InList of correlated sub-queries can be SymbolReference or other types
```

Comment:
Same, could you address my previous comment? https://github.com/prestodb/presto/pull/12747/commits/074162c1a55b0112606ba958cb1520ae35f10707#r284566495

Reply:
I believe `testDoubleNestedCorrelatedSubqueries` already covers the case? We cannot create such expression from parser, it is only created from sub query planner. Manually create an expression then test if it translates kind of beat its purpose?

Reply:
Added a hand written test on `TestSqlToRowExpressionTranslator`